core/sql/langman/LmRoutineJavaObj.cpp

```diff
@@ -192,6 +192,7 @@ LmResult LmRoutineJavaObj::invokeRoutineMethod(
           (*emitRowPtr_)(NULL,0,&qs);
         }
     }
+  jni->DeleteLocalRef(jniResult);
```

Comment:
Wait a minute... @sureshsubbiah, I think the spec you're referring to is the case where Java code calls a C++ routine, but in this particular fix scenario, we are doing the opposite, going from C++ to Java. So I'm not sure that the comment about "Local references are valid for the duration of a native method call. They are freed automatically after the native method returns" applies here.